src/aiida/cmdline/params/options/commands/setup.py

```diff
@@ -50,6 +50,8 @@ def get_profile_attribute_default(attribute_tuple, ctx):
     try:
         data = ctx.params['profile'].dictionary
         for part in parts:
+            if data is None:
+                return default
```

Comment:
Shouldn't this be outside of the for loop?

Reply:
No it is intentionally inside because `data` is iteratively updated in the for-loop, and the `None` value can occur later in the loop

Reply:
OK, then I misunderstood. I thought this was intended to capture `None` from the assignment above:
```python
        data = ctx.params['profile'].dictionary
```
Thanks for the clarification!